assets/Components.tsx

```diff
@@ -67,3 +67,14 @@ export const LargeInputField = styled.TextInput`
   padding: 10px;
   width: 317px;
 `;
+
+export const RootNavBackButton = styled.TouchableOpacity`
+  text-align: center;
+  align-items: center;
+  width: 20%;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 19px;
+  color: #525454;
```

Comment:
We shouldn't have any text styling as part of a styled button. Shift this into a styled Text container if required.